src/frameworks/flask/index.ts

```diff
@@ -13,7 +15,22 @@ export const type = FrameworkType.Framework;
 export async function discover(cwd: string) {
   if (!hasPipDependency("Flask", { cwd })) return;
   const results = await getDiscoveryResults(cwd).catch(() => undefined);
-  if (!results) return;
+  if (!results) {
+    logger.debug(
+      "Looks like you might be using Flask. Here're some tips that will help our tools work with your Python project:"
```

Comment:
Nit-level rephrase might be:

"Here are some tips that can help you make our tools work better with . . ."

or

"Here are some tips on using our our tools with your . . ."

Rationale: the tips don't help the tools, they help the tool-user.